src/main/java/com/coveros/selenified/Selenified.java

```diff
@@ -342,28 +341,30 @@ protected void startTest(Object[] dataProvider, Method method, ITestContext test
         }
         int invocationCount = (int) test.getAttribute(testName + INVOCATION_COUNT);
 
-        Browser myBrowser = browsers.get(invocationCount);
+        Capabilities capabilities = Selenified.capabilities.get(invocationCount);

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "capabilities" which hides the field declared at line 78. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AHiddenFieldCheck)